rust/libnewsboat-ffi/src/lib.rs

```diff
@@ -65,3 +65,13 @@ pub extern "C" fn rs_cstring_free(string: *mut c_char) {
         CString::from_raw(string);
     };
 }
+
+#[no_mangle]
+pub extern "C" fn rs_get_default_browser() -> *mut c_char {
+    let browser = utils::get_default_browser();
+    // Panic here can't happen because:
+    //1. get_default_browser() returns BROWSER environment variable 
```

Comment:
I don't think it explains anything. `CString` ["will return an error if the supplied bytes contain an internal 0 byte"}(https://doc.rust-lang.org/std/ffi/struct.CString.html#method.new), so the comment has to show that no zero bytes can be present in the string returned from `get_default_browser`.

I'd say that std::env::var returns an error if the variable contains invalid Unicode, and in that case `get_default_browser` will return "lynx", which obviously doesn't contain null bytes.